platform/darwin/src/MGLGeoJSONSource.h

```diff
@@ -6,6 +6,49 @@ NS_ASSUME_NONNULL_BEGIN
 
 @protocol MGLFeature;
 
+/**
+ The value of this option is an `NSNumber` object containing a boolean. If the data
+ is a collection of point features, setting this to true clusters the points by 
+ radius into groups. The default value is false.
```

Comment:
Nit: our convention is currently to use Objective-C Boolean values, so `NO` (in backticks) rather than `false`.